src/__tests__/components/forms/CreateSmsTemplate/__snapshots__/CreateSmsTemplate.test.tsx.snap

```diff
@@ -0,0 +1,1241 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CreateSmsTemplate component renders page 1`] = `
+<DocumentFragment>
+  <div
+    class="nhsuk-grid-row"
+  >
+    <div
+      class="nhsuk-back-link nhsuk-u-margin-bottom-6 nhsuk-u-margin-left-3"
+    >
+      <a
+        class="nhsuk-back-link__link"
+        href="/choose-a-template-type/undefined"
```

Comment:
why is this undefined here?

Reply:
Good spot. This is because `TemplateFormState.id` wasn't set on the `mockDeep<TemplateFormState>`. So the URL gets created as `undefined`. I've updated it now.